RecoLocalMuon/DTRecHit/plugins/SealModule.cc

```diff
@@ -10,9 +10,9 @@
 #include "RecoLocalMuon/DTRecHit/plugins/DTParametrizedDriftAlgo.h"
 
 #include "CalibMuon/DTDigiSync/interface/DTTTrigSyncFactory.h"
-#include "CalibMuon/DTDigiSync/src/DTTTrigSyncTOFCorr.h"
-#include "CalibMuon/DTDigiSync/src/DTTTrigSyncT0Only.h"
-#include "CalibMuon/DTDigiSync/src/DTTTrigSyncFromDB.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncTOFCorr.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncT0Only.h"
+#include "CalibMuon/DTDigiSync/interface/DTTTrigSyncFromDB.h"
```

Comment:
I think that it makes more sense to move `DEFINE_EDM_PLUGIN` to `CalibMuon/DTDigiSync` from this file, since the `DTTTrigSyncFactory` is defined there as well.

Reply:
agree @slava77. I am going to update this PR

Reply:
done